src/NuGet.Core/NuGet.Protocol/HttpSource/HttpSourceAuthenticationHandler.cs

```diff
@@ -96,11 +96,18 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
                 if (response.StatusCode == HttpStatusCode.Unauthorized ||
                     (configuration.PromptOn403 && response.StatusCode == HttpStatusCode.Forbidden))
                 {
-                    IList<Stopwatch> stopwatches = null;
+                    List<Stopwatch> stopwatches = null;
 
+#if NET5_0
+                    if (request.Options.TryGetValue(
+                        new HttpRequestOptionsKey<List<Stopwatch>>(HttpRetryHandler.StopwatchPropertyName),
+                        out stopwatches))
+                    {
+#else
                     if (request.Properties.TryGetValue(HttpRetryHandler.StopwatchPropertyName, out var value))
                     {
-                        stopwatches = value as IList<Stopwatch>;
+                        stopwatches = value as List<Stopwatch>;
```

Comment:
why did this change?

Reply:
I just wanted the types to match in the calls to `Set` and `TryGet`.